cmd/algoh/main.go

```diff
@@ -145,6 +145,7 @@ func main() {
 		}
 		err = cmd.Wait()
 		if err != nil {
+			captureErrorLogs(algohConfig, errorOutput, output, absolutePath, true)
```

Comment:
we want to capture the logs in all the exit cases, so let's do it correctly:
after the `err = cmd.Wait()`, add the following
```
defer func() {
                // capture logs if algod terminated prior to blockWatcher starting
		if !blockWatcherInitialized {
			captureErrorLogs(algohConfig, errorOutput, output, absolutePath, true)
		}
}
```
and remove the same section from below.
Also, in `reportErrorf`, make sure to call `Warnf` instead of `Fatalf`. We don't want to exit the process right away, ( the closing of the `done` channel would do that anyway, and we don't need the `algoh` stack trace.
There is a race condition around the `blockWatcherInitialized`, but I'm going to leave it for now.